### PR TITLE
feat: rating comp added

### PR DESCRIPTION
**Pull request by ShreyashMahajan**
Rating component for e-commerce is added

### CODE REVIEW

**Review by JantuDeb — APPROVED** [review]
This is nicely coded. I would suggest you to add screenshot or live preview

File: component_documentation/componentDoc.html [hunk 1]
```diff
@@ -1203,7 +1203,40 @@ <h2 class="modal__heading"> Heading</h2>
...
+      <!-- rating starts -->
 
+      <div class="component-container" id="modal--component">
+        <h1 class="component__heading">Rating</h1>
+        <p class="component__sub-heading">Rating is used to represent the profile pictures and icons. Monad UI has different sizes of Avatar.</p>
+        <div class="sub-component-container">
+            <h2 class="sub-component__heading">Rating</h2>
+           <div class="component-display">
+              <div class="rating-container">
+                  <img src="/image/star_yellow.png" class="rating__img" />
```
Comment (Priya-Patel31) [inline component_documentation/componentDoc.html]: fix : add alt tag to give some alternate text.

**Review by Priya-Patel31 — APPROVED** [review]
The code  is well written and really liked the way you are creating small PR for your component library . You can go ahead and merge this code . Add preview  link or screenshot .

File: component_documentation/componentDoc.html (hunk 1 above)
Comment (ShreyashMahajan) [reply component_documentation/componentDoc.html]: ok, ty i will add it

**Comment by ShreyashMahajan** [comment]
> This is nicely coded. I would suggest you to add screenshot or live preview

Thank you, i will add live preview from in next pr

**Pull request merged**